lib/v2/ieee-security/sp.js

```diff
@@ -0,0 +1,35 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const url = 'https://www.ieee-security.org/';
+// https://www.ieee-security.org/TC/SP2023/program-papers.html
+
+module.exports = async (ctx) => {
+    const last = new Date().getFullYear() + 2;
+    const urlList = Array.from({ length: last }, (_, v) => `${url}TC/SP${v + 2020}/program-papers.html`);
```

Comment:
```suggestion
    const last = new Date().getFullYear() + 1;
    const urlList = Array.from({ length: last - 2020 }, (_, v) => `${url}TC/SP${v + 2020}/program-papers.html`);
```
This will generate urls from 2020 to 2022